frontend/src/components/TimeEntry/EditEntryForm.js

```diff
@@ -42,17 +41,16 @@ const initialValues = (defaultDate, entry, timeSheetId) => {
   const newEntry = {
     ...entry,
     startDateTime: moment.utc(entry.startDateTime),
-    endDateTime: moment.utc(entry.endDateTime)
+    numberOfHours: entry.numberOfHours
   };
-  const numberOfHours = computeNumberOfHours(newEntry.startDateTime, newEntry.endDateTime);
+  const numberOfHours = new Array(newEntry.numberOfHours);
```

Comment:
computeNumberOfHours return a number. 
`numberOfHours = new Array(newEntry.numberOfHours);` is not used, you can remove it

Reply:
ok. changed the code. Thanks for reporting it.